js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/Sidebar.tsx

```diff
@@ -347,7 +349,35 @@ export const AssetGraphExplorerSidebar = React.memo(
           />
         </Box>
         <div>
-          <Container ref={containerRef}>
+          <Container
+            ref={containerRef}
+            onKeyDown={(e) => {
+              let nextIndex = 0;
+              if (e.code === 'ArrowDown' || e.code === 'ArrowUp') {
+                nextIndex = indexOfLastSelectedNodeRef.current + (e.code === 'ArrowDown' ? 1 : -1);
+                indexOfLastSelectedNodeRef.current = nextIndex;
+                e.preventDefault();
+                const nextNode = renderedNodes[nextIndex % renderedNodes.length]!;
```

Comment:
Hmm I think there's a missing useCallback in here -- right now there's no reason to put the indexOfLastSelectedNode into a ref because this onKeyDown is re-generated on every render. (And I think that's totally fine, I'd maybe get rid of the ref?)

I also think that this doesn't handle stopping at the top or bottom of the list quite right, because the mod operator on this line doesn't address the desired `0 - 1 => 0` case (`-2 % 5 = -2`).  I think it handles the positive bound case, but the negative bound case requires adding `renderedNodes.length` to nextIndex before modding it.

I think for safety sake it'd be nice to remove the bang operator here and handle the null with an early return, just so that there's no chance this crashes